Makefile

```diff
@@ -250,9 +250,8 @@ define CPU_TF27_TAGS
 endef
 endif
 
-TORCH_VERSION := 1.12
-export CPU_TF27_ENVIRONMENT_NAME := $(CPU_PREFIX)pytorch-$(TORCH_VERSION)-tf-2.7$(CPU_SUFFIX)
-export GPU_TF27_ENVIRONMENT_NAME := $(CUDA_112_PREFIX)pytorch-$(TORCH_VERSION)-tf-2.7$(GPU_SUFFIX)
+export CPU_TF27_ENVIRONMENT_NAME := $(CPU_PREFIX)-tf-2.7$(CPU_SUFFIX)
```

Comment:
This results in `docker pull determinedai/environments-dev:cuda-11.2--tf-2.7-gpu-mpi-0.20.1` (note the double dash). I think (CPU_PREFIX) already contains a trailing dash.

Reply:
See: https://hub.docker.com/r/determinedai/environments-dev/tags?page=1&name=tf-2.7

Reply:
Oops, sorry. I should have seen that.